Efficiency/GetEfficiencyB0.py

```diff
@@ -0,0 +1,138 @@
+'''
+Script for the computation of the B0 meson efficiency
+run: python GetEfficiencyB0.py configFileName.yml cutSetFileName.yml
+'''
+
+import ROOT 
+import pandas as pd 
+import numpy as np
+import argparse
+import yaml
+import ctypes
+import sys
+sys.path.append('Utils')
+from DfUtils import read_parquet_in_batches
+from AnalysisUtils import EvaluateEfficiencyFromHistos
+
+def ComputeEfficiency(configFileName, cutSetFileName):
+    """
+    Compute the efficiency of a B0 particle based on the given configuration and cut set.
+
+    Args:
+        configFileName (str): The file name of the configuration file.
+        cutSetFileName (str): The file name of the cut set file.
+
+    Returns:
+        None
+    """
+    
+    with open(args.cutSetFileName, 'r') as ymlcutsetFile:
+        cutset = yaml.safe_load(ymlcutsetFile)
+
+    with open(args.configFileName, 'r') as ymlconfigFile:
+        config = yaml.safe_load(ymlconfigFile)
+
+    # Set style
+    ROOT.gStyle.SetOptStat(0)
+    ROOT.gStyle.SetPadLeftMargin(0.14)
+    ROOT.gStyle.SetPadBottomMargin(0.12)
+    ROOT.gStyle.SetPadTopMargin(0.05)
+    ROOT.gStyle.SetPadRightMargin(0.05)
+    ROOT.gStyle.SetTitleOffset(1.2, 'X')
+    ROOT.gStyle.SetPadTickX(1)
+    ROOT.gStyle.SetPadTickY(1)
+
+    ptMins = cutset['pt']['mins']
+    ptMaxs = cutset['pt']['maxs']
+    ptLims = list(ptMins)
+    nPtBins = len(ptMins)
+    ptLims.append(ptMaxs[-1])
+
+    hRecoIntegrated = ROOT.TH1F('hRecoIntegrated', ';#it{p}_{T} (GeV/#it{c});Reconstructed', nPtBins, np.asarray(ptLims, 'd'))
+    hGenIntegrated = ROOT.TH1F('hGenIntegrated', ';#it{p}_{T} (GeV/#it{c});Generated', nPtBins, np.asarray(ptLims, 'd'))
+
+    for iFile, inFileName in enumerate(config['gen']['fileNames']):
+        inFile = ROOT.TFile.Open(inFileName)
+        hSparseGen = inFile.Get(config['gen']['sparseName'])
+        if iFile == 0:
+            hGen = hSparseGen.Projection(config['gen']['ptAxis'])
+            hGen.SetDirectory(0)
+        else:
+            hGen.Add(hSparseGen.Projection(config['gen']['ptAxis']))
+        inFile.Close()
```

Comment:
It would also be nice to look at the acceptance factor separately (in the output of the task we also have the sparse with the generated B0 mesons having the daughters in acceptance). We can add it in a second iteration